package/base-files/files/lib/functions/system.sh

```diff
@@ -15,7 +15,11 @@ get_mac_binary() {
 get_mac_label() {
 	local basepath="/proc/device-tree"
 	local macdevice="$(cat "$basepath/aliases/label-mac-device" 2>/dev/null)"
-	[ -n "$macdevice" ] && get_mac_binary "$basepath/$macdevice/mac-address" 0 2>/dev/null
+	local macaddr
+
+	[ -n "$macdevice" ] && macaddr=$(get_mac_binary "$basepath/$macdevice/mac-address" 0 2>/dev/null)
```

Comment:
If you take a [look at the current](https://elixir.bootlin.com/linux/latest/source/Documentation/devicetree/bindings/net/ethernet.txt) DT binding docs, it might be as well in `local-mac-address`.

Reply:
I'm aware of that, but I never encountered an example where local-mac-address was present and mac-address was not. However, one could extend it for "possible future use". I'm agnostic here, so shall I do so?

Reply:
I will just add it, it doesn't hurt and will cover more possible locations.